src/components/strategies/create/CreateStrategyTypeMenu.tsx

```diff
@@ -183,12 +183,14 @@ export const CreateStrategyTypeMenu: FC<UseStrategyCreateReturn> = ({
       <Link
         {...selectedStrategySettings}
         params={{}}
-        className={cn(
-          buttonStyles({ variant: 'success', fullWidth: true, size: 'lg' }),
-          selectedStrategySettings
-            ? ''
-            : 'pointer-events-none cursor-not-allowed opacity-40'
-        )}
+        className={`
+          ${buttonStyles({ variant: 'success', fullWidth: true, size: 'lg' })}
+          ${
+            selectedStrategySettings
+              ? ''
+              : 'pointer-events-none cursor-not-allowed opacity-40'
+          }
+        `}
```

Comment:
buttonStyles applies the same opacity and cursor-not-allowed, so I think it's fine not to use cn(), please confirm that's intended